pkg/k3s/preinstall/preinstall.go

```diff
@@ -50,7 +50,7 @@ func FilesDownloadHTTP(mgr *manager.Manager, filepath, version, arch string) err
 		helm.GetCmd = mgr.DownloadCommand(helm.Path, helm.Url)
 	} else {
 		etcd.Url = fmt.Sprintf("https://github.com/coreos/etcd/releases/download/%s/etcd-%s-linux-%s.tar.gz", etcd.Version, etcd.Version, etcd.Arch)
-		k3s.Url = fmt.Sprintf("https://github.com/rancher/k3s/releases/download/%s+k3s1/k3s", k3s.Version)
+		k3s.Url = fmt.Sprintf("https://github.com/k3s-io/k3s/releases/download/v1.21.4+k3s1/k3s-arm64", k3s.Version)
```

Comment:
Hello! Maybe that is not graceful. What happend if we want to install a **v1.22.1** k3s cluster?

Reply:
Hey sorry missed the version, i have made the changes